frontend/src/metabase-types/api/dashboard.ts

```diff
@@ -54,7 +54,7 @@ export type DashCardId = number;
 export type BaseDashboardCard = {
   id: DashCardId;
   dashboard_id: DashboardId;
-  dashboard_tab_id?: DashboardTabId;
+  dashboard_tab_id?: DashboardTabId | null;
```

Comment:
Can we narrow it down to `dashboard_tab_id: DashboardTabId | null;`?

Reply:
Good point 👍 Fixed in [`2ba9048`](https://github.com/metabase/metabase/pull/38447/commits/2ba90489dc1b3d1617306b18fe12c1d9a6537703)